client/components/student/student.service.js

```diff
@@ -7,8 +7,7 @@ app.factory('Student', function($resource) {
     id: '@_id'
   }, {
     list: {
-      method: 'GET',
-      isArray: true
+      method: 'GET'
```

Comment:
Ok I'm removing this in the next commit. I won't touch the server controller as that can be modified in another change-list.